src/main/java/uk/gov/pay/api/model/CreateCardPaymentRequest.java

```diff
@@ -136,6 +137,8 @@ public String getReturnUrl() {
     }
 
     @JsonProperty("metadata")
+    @ApiModelProperty(value = "Additional metadata - up to 10 key-value pairs - on the payment. Each key must be between 1 and 30 characters long. Values must be no greater than 50 characters long and be of types String, Boolean or Number.",
```

Comment:
The 50 character limit only applies if it’s a string value. Also, https://json.org/ calls them “name/value” pairs (rather than “key-value” pairs, so perhaps we should use that terminology and apparently “string” and “number” and the like don’t start with capital letters in JavaScriptland. Might be useful to mention it’s an object too. So maybe tweak it to be something like this: “Additional metadata — object containing up to 10 name/value pairs. Each name must be between 1 and 30 characters long. Each value must be either a string (no greater than 50 characters long), boolean or number.”